birdhouse/pavics-compose.sh

```diff
@@ -157,44 +95,47 @@ done
 export AUTODEPLOY_EXTRA_REPOS_AS_DOCKER_VOLUMES
 
 # we apply all the templates
-find ./config ./components ./optional-components $EXTRA_CONF_DIRS -name '*.template' |
+find $ALL_CONF_DIRS -name '*.template' |
```

Comment:
This `ALL_CONF_DIR` should not contain the `deployment/` folder so `cron` file would not need to be in a `.gitignore` file earlier.

Reply:
If you had kept the same `find ./config ./components ./optional-components $EXTRA_CONF_DIRS`, then `thredds` config files will always be template expanded for `testthredds` to use, regardless if `thredds` is enabled or not :D

I think there is no harm in always template expand everything since all the `.gitignore` are already properly setup.